modules/standard/Random.chpl

```diff
@@ -1068,6 +1052,27 @@ module Random {
     }
 
 
+    /*
+
+     Low-level PCG RNG implementation.
+
+     Besides a Chapel :class:`RandomStream` interface, the :mod:`PCGRandomLib`
```

Comment:
I think I'm going to leave it as PCGRandomLib as PCGRandomUtil/Help/LowLevel seem worse to me.
I think we're going to need a strategy for naming such things and that we should change PCGRandomLib to use that strategy once we know what it is.